### ISSUE TITLE
Question about custom modes

### ISSUE THREAD

**Issue by escoand** [rospogrigio/localtuya#1125]
My installes tuya climate devices use a strange variant of HVAC modes.
I would understand if it would be denied in master because it's so different.

```python
HVAC_MODE_SETS = {
    "cold/hot/wind": {
        HVAC_MODE_HEAT: "cold",
        HVAC_MODE_AUTO: "hot",
        // "wind" means temporary manual
    },
}
```

But anyway, wouldn't it be better to have a input field allowing to insert such specific cases? So we don't have to clutter the master code?

**Comment by escoand** [comment]
By the way I have Beok Tol47Wifi:
https://www.beok-controls.com/room-thermostat/underfloor-heating-thermostat/wi-fi-underfloor-heating-thermostat.html

**Comment by lazmo88** [comment]
@escoand I agree that there should be easier way to configure edge cases. I am in a similar situation where device called "PTC-2021" class "qn" is missing most of the functions.

By debugging the device, it seems the modes are "normal/auto" where normal = manual mode.

My device also has 3 levels of heating: fan only, 1kw and 2kw modes.

Additionally it has a support for swing mode (on/off) and window check (true/false).

Unless I am missing something, the GUI route doesn't allow me to config this device.

Is there a yaml way to config custom devices or am I out of luck?
